strings/salesman_replacement.json

```diff
@@ -0,0 +1,49 @@
+{
+
+    "salesman": {
+        "shuttle": "cungadero",
+        "clam": "pipis",
+        "clams": "pipis",
+        "egg": "pipis",
+        "eggs": "pipis",
+        "love": "hyperlink blocked",
+        "captain": "big shot",
+        "dollar": "kromer",
+        "dollars": "kromer",
+        "credit": "kromer",
+        "credits": "kromer",
+        "basement": "deep abyss",
+        "mech": "workout-ready body",
+        "exosuit": "workout-ready body",
+        "phazon": "NEO",
```

Comment:
the mech replacements are actually pretty good, i'd just make any mecha turn into this so it hits more on replacement and gives roleplayers:tm: options

Reply:
I made the Phazon specifically be the NEO so that spamton roleplayers would have a sort-of goal, in the form of the hardest to obtain mech.